BSPConvert.Lib/Source/EntityConverter.cs

```diff
@@ -797,14 +797,14 @@ private void FireTargetGiveOnOutput(Entity entity, Entity targetGive, string out
 				switch (target.ClassName)
 				{
 					case "item_haste":
-						SetHasteOnOutput(entity, ConvertPowerupCount(target["count"]), output, delay);
+						SetHasteOnOutput(entity, ConvertPowerupCount(target["count"]), output, delay, false);
```

Comment:
You can just add 0.008 to the delay here instead of adding a new parameter to `SetHasteOnOutput`